src/main/CMV.java

```diff
@@ -63,7 +65,25 @@ private Boolean lic1_calculate() {
     }
 
     
-    private Boolean lic2_calculate() {
+    private boolean lic2_calculate() {
+        final double EPSILON = Parameters.EPSILON;
+        if (EPSILON < 0 || EPSILON >= Math.PI){
+            return false;
```

Comment:
I mean, these are just requirements on the parameters, it doesn't say that it should return false in such a case

Rather it should perhaps be a crash or something because if the epsilon is bigger than pi it would imply negative angles

Maybe we should add all these conditions to the parameters class?

Reply:
Not a bad shout at all! will write it in the discord